### PR TITLE
docs: expand README

### LINKED ISSUE #68: Write a README
@eaball35 mentioned an actual README would have been very helpful!

### PR DESCRIPTION
**Pull request by tbpg**
Fixes #68.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+```
+
+## Usage
+
+There are two steps for uploading docs:
+1. Create a metadata file. See [`metadata.proto`](./docuploader/protos/metadata.proto)
+   for the format.
+   * You can use `docuploader create-metadata` to create a `docs.metadata` file.
+   * Alternatively, you can create a `docs.metadata.json` file independently.
+1. Upload the docs with `docuploader upload`.
```
Comment (dandhlee) [inline README.md:23]: Most of the time we use this it it followed by a lot of options. But others could be simply using `docuploader upload .`, what do you think about adding the usage notes?

```
Usage: docuploader upload [OPTIONS] DOCUMENTATION_PATH

Options:
  --staging-bucket TEXT      The bucket to upload the staged documentation to.
  --credentials TEXT         Path to the credentials file to use for Google
                             Cloud Storage.

  --metadata-file TEXT       Path to the docs.metadata file.
  --destination-prefix TEXT  Prefix to include when uploading tar file. A -
                             will be added after the prefix, if there is one.

  --help                     Show this message and exit.
```

**Review by dandhlee — APPROVED** [review]
LGTM, just a minor comment below. Happy to submit as-is.

File: README.md (hunk 1 above)
Comment (eaball35) [reply README.md:23]: I agree - providing all the possible options like this would be helpful

**Review by eaball35 — APPROVED** [review]
Left comment but otherwise LGTM!

File: README.md (hunk 1 above)
Comment (tbpg) [reply README.md]: Done.

**Review by tbpg — COMMENTED** [review]
Thanks!

**Pull request merged**